lib/brainstem/presenter.rb

```diff
@@ -28,6 +28,32 @@ def self.default_sort_order(sort_string = nil)
       end
     end
 
+    # @overload exclude_count
+    #   set exclude_count = true and allow_count = false (i.e. exclude the count and do not
+    #   do not allow a request for the count via the url query string)
+    #   @return [Boolean, Boolean]
+    # @overload exclude_count( allow: true )
+    #   set exclude_count = true and allow_count = false (i.e. exclude the count by default
+    #   but allows a request for it via the url query string with (e.g. ?count=true)
+    #   @return [Boolean, Boolean]    
+    # @overload exclude_count( exclude: false )
+    #   set exclude_count = false. This is only needed for testing to toggle back to the
+    #   including the count by default, otherwise some of the subsequent tests may fail
+    #   @return [Boolean, Boolean]
+    def self.exclude_count(exclude: true, allow: false)
```

Comment:
Brainstem still needs to support 1.9, so kwargs can't be used quite yet unfortunately.